examples/Animometer.cpp

```diff
@@ -22,6 +22,8 @@
 nxt::Device device;
 nxt::Queue queue;
 nxt::Pipeline pipeline;
+nxt::RenderPass renderpass;
```

Comment:
I put just the setup in Utils `CreateDefaultRenderPass` to initialize a render pass and framebuffer.